NineChronicles.Headless/GraphTypes/States/Models/MeadPledgeType.cs

```diff
@@ -0,0 +1,15 @@
+using GraphQL.Types;
+using Libplanet;
+using Libplanet.Explorer.GraphTypes;
+
+namespace NineChronicles.Headless.GraphTypes.States.Models;
+
+public class MeadPledgeType : ObjectGraphType<(Address?, bool, int)>
```

Comment:
It will be better to use meaningful variable names. How about using named-tuple to help understand?

```suggestion
public class MeadPledgeType : ObjectGraphType<(Address? Address, bool Approved, int Mead)>
```